content/operate/kubernetes/release-notes/7-4-2-12.md

```diff
@@ -0,0 +1,163 @@
+| **Azure AKS** |  |  | <span title="Deprecated" class="font-serif">:warning:</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| **Google GKE** |  | <span title="Deprecated" class="font-serif">:warning:</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| **Rancher RKE2** | <span title="X icon">&#x274c;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |
+| **VMware TKG 2.2** | <span title="Deprecated" class="font-serif">:warning:</span> | <span title="Deprecated" class="font-serif">:warning:</span> |  |  |  |  |
+| **VMware TKG 2.3** | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |  |
+| **VMware TKG 2.4** |  | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |
+| **OpenShift version** | **4.11** | **4.12** | **4.13** | **4.14** | **4.15** |  |
+|  | <span title="Deprecated" class="font-serif">:warning:</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |
+| **VMware TKGI version** | **1.15** | **1.16** | **1.17** | **1.18** |  |  |
+|  | <span title="X icon">&#x274c;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |
+
+
+## Downloads
+
+- **Redis Enterprise**: `redislabs/redis:7.4.2-129`
+- **Operator**: `redislabs/operator:7.4.2-12`
+- **Services Rigger**: `redislabs/k8s-controller:7.4.2-12`
+
+### OpenShift images
+
+- **Redis Enterprise**: `registry.connect.redhat.com/redislabs/redis-enterprise:7.4.2-129.rhel8-openshift`
+- **Operator**: `registry.connect.redhat.com/redislabs/redis-enterprise-operator:7.4.2-12`
+- **Services Rigger**: `registry.connect.redhat.com/redislabs/services-manager:7.4.2-12`
+
+### OLM bundle
+
+**Redis Enterprise operator bundle** : `v7.4.2-12.0`
+
+## Known limitations
+
+### New limitations
+
+- **Missing endpoint for admission endpoint (rare) (RED-119469)** Restart the operator pod.
+
+### Existing limitations
+
+- **The REDB “redisVersion” field can’t be used for memcached databases(RED-119152)**
+
+- **When modifying the database suffix for an Active-Active database, while the service-rigger is in a terminating state, the services-rigger will delete and create the ingress or route resources in a loop (RED-107687)** Wait until the services- rigger pod has finished to terminate it.
```

Comment:
Space between "services-" and "rigger" in the last occurrence.

Reply:
Highlights: This release has many enhancements, most notably support for persistent volume expansion in the REC. Also, some essential changes in module handling support newer capabilities prompted by changes in the underlying Redis Enterprise around versioning. 

Please change the legend at the top of the release notes and the supported distribution pages from "Deprecated" to "Deprecation warning". We changed this as well on RS for clarity.

The table in the release notes for the supported distributions seems to be misalgned on the K8s version and the columns below.